File: api-python/src/shared/openai/types/chat_completion_response.py
from pydantic import BaseModel, Field
from shared.openai.types.chat_completion_choice import ChatCompletionChoice
from shared.openai.types.usage import Usage


class ChatCompletionResponse(BaseModel):
    id: str
    object: str = Field(default="chat.completion")
    created: int
    model: str
    choices: list[ChatCompletionChoice]
    usage: Usage

File: api/real-time-ai-agent/agent/src/agent/types/chat_completion_response.rs
use serde::Serialize;

use crate::agent::types::chat_completion_choice::ChatCompletionChoice;

#[derive(Serialize)]
pub struct ChatCompletionResponse {
    pub id: String,
    pub object: String,
    pub created: i64,
    pub model: String,
    pub choices: Vec<ChatCompletionChoice>,
}
